clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10025,6 +10025,11 @@ def err_duplicate_case_differing_expr : Error<
 def warn_case_empty_range : Warning<"empty case range specified">;
 def warn_missing_case_for_condition :
   Warning<"no case matching constant switch condition '%0'">;
+def err_loop_attr_duplication : Error<
+  "duplicate loop attribute %0">;
+def err_attribute_power_of_two_in_range : Error<
+  "%0 attribute requires an integer argument which is a constant power of two "
+  "between %1 and %2 inclusive; got %3">;
```

Comment:
we don't typically do something like `got %3` here?  I've at least not seen us do that before.  If we wish to provide it (as perhaps the person might get confused after constant evaluation?), it should probably be `provided argument was %3`

Reply:
>>I've at least not seen us do that before. If we wish to provide it (as perhaps the person might get confused after constant evaluation?), it should probably be provided argument was %3

It makes sense to me. Thanks for the suggestion.